applications/nrf_desktop/README.rst

```diff
@@ -3,2385 +3,21 @@
-More information about each application module and its configuration details is available on the subpages.
-
-Each module documentation page has a table that shows the relations between module events.
-`Module event tables`_ for some modules include extensive lists of source and sink modules.
-These are valid for events that have many listeners or sources, and are gathered on the :ref:`nrf_desktop_event_rel_modules` subpage.
+See the subpages for detailed documentation on the application and its modules:
 
 .. toctree::
    :maxdepth: 1
    :caption: Subpages:
 
-   doc/main.rst
-   doc/battery_charger.rst
-   doc/battery_meas.rst
-   doc/ble_adv.rst
-   doc/ble_bond.rst
-   doc/ble_conn_params.rst
-   doc/ble_discovery.rst
-   doc/ble_latency.rst
-   doc/ble_passkey.rst
-   doc/ble_qos.rst
-   doc/ble_scan.rst
-   doc/ble_state_pm.rst
-   doc/ble_state.rst
-   doc/board.rst
-   doc/buttons.rst
-   doc/buttons_sim.rst
-   doc/click_detector.rst
-   doc/config_channel.rst
-   doc/cpu_meas.rst
-   doc/dev_descr.rst
-   doc/dfu.rst
-   doc/dfu_mcumgr.rst
-   doc/dfu_lock.rst
-   doc/factory_reset.rst
-   doc/failsafe.rst
-   doc/fast_pair_app.rst
-   doc/fn_keys.rst
-   doc/bas.rst
-   doc/hid_forward.rst
-   doc/hid_state.rst
-   doc/hid_state_pm.rst
-   doc/hids.rst
-   doc/info.rst
-   doc/led_state.rst
-   doc/led_stream.rst
-   doc/leds.rst
-   doc/motion.rst
-   doc/passkey.rst
-   doc/power_manager.rst
-   doc/nrf_profiler_sync.rst
-   doc/qos.rst
-   doc/selector.rst
-   doc/smp.rst
-   doc/settings_loader.rst
-   doc/swift_pair_app.rst
-   doc/usb_state_pm.rst
-   doc/usb_state.rst
-   doc/watchdog.rst
-   doc/wheel.rst
-   doc/constlat.rst
-   doc/hfclk_lock.rst
-   doc/event_rel_modules.rst
-
-.. _config_desktop_app_options:
-
-Application configuration options
-*********************************
-
-Following are the application specific configuration options that can be configured for the nRF desktop and the internal modules:
-
-.. options-from-kconfig::
-   :show-type:
-
-API documentation
-*****************
-
-Following are the API elements used by the application.
-
-HID reports
-===========
-
-| Header file: :file:`applications/nrf_desktop/configuration/common/hid_report_desc.h`
-| Source file: :file:`applications/nrf_desktop/configuration/common/hid_report_desc.c`
-
-.. doxygengroup:: nrf_desktop_hid_reports
-   :project: nrf
-   :members:
-
-LED states
-==========
-
-| Header file: :file:`applications/nrf_desktop/configuration/common/led_state.h`
-| Source file: :file:`applications/nrf_desktop/src/modules/led_state.c`
-
-.. doxygengroup:: nrf_desktop_led_state
-   :project: nrf
-   :members:
-
-USB events
-==========
-
-| Header file: :file:`applications/nrf_desktop/src/events/usb_event.h`
-| Source file: :file:`applications/nrf_desktop/src/modules/usb_state.c`
-
-.. doxygengroup:: nrf_desktop_usb_event
-   :project: nrf
-   :members:
+   nrf_desktop_description
+   nrf_desktop_integration
+   modules
```

Comment:
Consider calling files  `nrf_desktop_modules`, `nrf_desktop_api`, `nrf_desktop_configuration_option` (to ensure consistent file naming scheme. Alternatively maybe we can remove `nrf_desktop` prefix from these file names